UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -1055,4 +1056,11 @@ public void RpcShowFaithSelectScreen(NetworkConnectionToClient target)
 	{
 		UIManager.Instance.ChaplainFirstTimeSelectScreen.gameObject.SetActive(true);
 	}
+
+	[ConsoleMethod("free-faith-points", "Awards 500 free faith points."), Command]
+	public void CmdFreeFaithPoints()
```

Comment:
Can't this just be triggered by the player?

Reply:
it cant be trigger by other players due to it requiring the owner to call it
tho if you mean can the owner player can call this on his own, isn't that technically what all the console commands do?
The admin check on them are entirely client based, which can be skipped

Reply:
oh? which console commands?